docs/package_templates/cmake_package/all/conanfile.py

```diff
@@ -114,7 +100,10 @@ def generate(self):
         tc.cache_variables["PACKAGE_CUSTOM_DEFINITION"] = True
         if is_msvc(self):
             # don't use self.settings.compiler.runtime
-            tc.cache_variables.definitions["USE_MSVC_RUNTIME_LIBRARY_DLL"] = "MD" in msvc_runtime_flag(self)
+            tc.cache_variables.definitions["USE_MSVC_RUNTIME_LIBRARY_DLL"] = not is_msvc_static_runtime(self)
+        # deps_cpp_info, deps_env_info and deps_user_info are no longer used
+        if self.dependencies["dependency"].options.foobar:
+            tc.cache_variables["DEPENDENCY_LIBPATH"] = self.dependencies["dependency"].cpp_info.libdirs
```

Comment:
Here I would use something more straightforward. If we add here things related to paths it looks like Conan is not managing them:

```suggestion
        tc.cache_variables["DEPENDENCY_OPTION_FOOBAR"] = self.dependencies["dependency"].options.foobar
```

Reply:
Yes, makes more sense. Dependency path was the first thing which came to my mind.